doc/source/dev/contributor/rendering_documentation.rst

```diff
@@ -0,0 +1,72 @@
+.. _rendering-documentation:
+
+===================================
+Rendering Documentation with Sphinx
+===================================
+
+SciPy docstrings are rendered to html using `Sphinx`_. Writing
+docstrings is covered in the `numpydoc docstring guide`_; this document
+explains how to check that docstrings render properly.
+
+*For a video walkthrough, please see* \ `Rendering SciPy Documentation
+with Sphinx`_ \ *.*
+
+.. _rendering-documentation-locally:
+
+Rendering Documentation Locally
+-------------------------------
+
+To render the documentation on your own machine (macOS or Linux):
+
+#. Install `Sphinx`_ or ensure that your installation is up to date.
+#. In a terminal window, browse to the ``scipy/doc`` directory. Note the
+   presence of the file ``Makefile``.
+#. If this is your first time building the docs, execute ``git submodule
+   init``. After you’ve initialized for the first time, enter ``git submodule
+   update`` instead. Some of the documentation theme files are not distributed
+   with the main ``scipy`` repository; this keeps them up to date with
+   `git submodules`_.
+#. Enter ``make html-scipyorg``. This uses the `Make build automation tool`_
+   to execute the documentation build instructions from the ``Makefile``.
+   This can take a while the first time, but subsequent documentation builds
+   are typically much faster. *Note: If you use a virtual environment for
+   development, activate it first.*
+#. View the documentation in ``scipy/doc/build/html-scipyorg``. You can start
+   with ``index.html`` and browse, or you can jump straight to the file you’re
+   interested in.
+
+.. note::
+
+   Changes to certain documents do not take effect when Sphinx documentation
```

Comment:
Do we know what kinds of changes exhibit this behavior, or is it seemingly arbitrary?

Reply:
I know it happens when updating documentation of solver methods, but I'm not sure if that's the only time, so I didn't speculate. If someone else has an answer it would be good to know.